pedsnetdcc/__init__.py

```diff
@@ -10,3 +37,13 @@
     'relationship',
     'source_to_concept_map'
 )
+
+DATA_MODELS_SERVICE = 'http://data-models-service.research.chop.edu/'
+
+from pedsnetdcc.age_transform import AgeTransform
+from pedsnetdcc.concept_name_transform import ConceptNameTransform
+from pedsnetdcc.site_name_transform import SiteNameTransform
+
+TRANSFORMS = (AgeTransform, ConceptNameTransform, SiteNameTransform)
+
+__all__ = (__version__, VOCAB_TABLES, DATA_MODELS_SERVICE, TRANSFORMS)
```

Comment:
The purpose of `__all__` is to only expose a controlled list of attributes if a user lives dangerously and performs `from pedsnet import *`.  Since I imported the transform classes to build the `TRANSFORMS` list, I didn't necessarily want to expose those also, so I just reflexively used `__all__`.

It's not really needed for us, since the main reasons for using it don't apply.  This is not a general-purpose public API, and we have no need or inclination to import `*`.  Plus, it imposes a maintenance burden.